save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/TabMenuBar.kt

```diff
@@ -0,0 +1,58 @@
+package com.saveourtool.save.entities.benchmarks
+
+import com.saveourtool.save.domain.Role
+
+/**
+ * Interface for tab bar in many pages
+ */
+interface TabMenuBar<T> {
+    /**
+     * Default value in every Enum classes
+     */
+    val defaultTab: T
+
+    /**
+     * Regular expression for url classification
+     */
+    val regexForUrlClassification: Regex
+
+    /**
+     * Pair consisting of a short address of the default tab and a long prefix of the addresses of all other tabs
```

Comment:
Sounds very complicated, please create a data class with properly named fields if such structure is really necessary